.github/workflows/windows.yml

```diff
@@ -0,0 +1,411 @@
+name: windows
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - '[0-9]+.[0-9]{2}'
+    tags:
+      - 'v*'
+  pull_request:
+
+env:
+  GO_VERSION: 1.18
```

Comment:
Perhaps use the same minor release

Reply:
done, also disabled github annotations for codecov